responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -675,6 +675,7 @@ def _load(path, rai_insights):
 
         inst.__dict__[CounterfactualManager._COUNTERFACTUAL_CONFIG_LIST] = []
 
+        DirectoryManager.ensure_dir_exists(path)
         all_cf_dirs = DirectoryManager.list_sub_directories(path)
```

Comment:
Ideally the call to list_sub_directories() should return an empty list. Why do you need to create a directory?

Reply:
This seemed the lowest-risk thing to do - normally there would be a directory there. Also suppose a user then adds an insight via the manager - will that automatically create the directory if it isn't there?

Reply:
Although, a new save would presumably be a whole new directory tree. Let me see if adjusting the directory list to be empty works.

Reply:
It works, and on reflection is probably a better solution. Have adjusted the PR.